boards/arm/lpcxpresso11u68/support/lpc_checksum_README

```diff
@@ -0,0 +1,40 @@
+lpc_checksum
+############
+
+Origin:
+   https://github.com/basilfx/lpc_checksum
+
```

Comment:
Instead of including the script within the zephyr repo, could we just add it as an (optional) dependency to be installed using `pip install lpc_checksum`?

Reply:
Yes we can :) When I looked in scripts/requirements*, I missed the "optional" part. Is that possible to install the script only when building Zephyr for a LPC11U6x MCU ? It was not feeling right to me to install this script in everybody's $HOME. That's the reason we chose the direct import. But if a dependency is the right way to go, please confirm and I'll do the change.

Reply:
What do you think @henrikbrixandersen ?

Reply:
I think we should add the dependency to either `scripts/requirements-base.txt` or `scripts/requirements-extras.txt`.

Reply:
The dependency has been added to scripts/requirements-extra.txt